pkg/pool-weighted/contracts/smart/InvestmentPool.sol

```diff
@@ -46,20 +58,25 @@ contract InvestmentPool is BaseWeightedPool, ReentrancyGuard {
     // [ 155 bits|   5 bits |  32 bits   |   64 bits    |
     // [ unused  | decimals | end weight | start weight |
     // |MSB                                          LSB|
-    mapping(IERC20 => bytes32) private _poolState;
+    mapping(IERC20 => bytes32) private _tokenState;
 
     uint256 private constant _START_WEIGHT_OFFSET = 0;
     uint256 private constant _END_WEIGHT_OFFSET = 64;
     uint256 private constant _DECIMAL_DIFF_OFFSET = 96;
 
-    // Time travel comment
-    // [ 192 bits | 32 bits  |  32 bits   |
-    // [  unused  | end time | start time |
-    // |MSB                            LSB|
-    //bytes32 private _gradualUpdateTimestamps;
+    // Adding or removing liquidity while paused must be proportional
+    // Proportional join is defined as one where the BPT prices are all similar
+    uint256 private constant _PROPORTIONAL_TOLERANCE_PCT = 1e16; // 1%
```

Comment:
After reviewing the mechanics of how this threshold is imposed, I'd recommend shrinking it to 0.1% or lower.

Reply:
Sounds good. Made the test tighter so it still works.